### PR TITLE
Update French translations

### PR DESCRIPTION
**Pull request by Vlady19**

### CODE REVIEW

File: res/translations/fr/messages.ftl [hunk 1]
```diff
@@ -96,8 +96,7 @@ configuration_dialog_button_cancel = Annuler
 running_title = En cours
 running_node_title = {$chain_name} Nœud de la blockchain
 running_node_title_tooltip = Cliquez pour ouvrir dans le gestionnaire de fichiers
-# TODO: Translate
-running_node_connections_tooltip = {$connected_peers}/{$expected_peers} peers connected, click for details about required P2P ports
+running_node_connections_tooltip = {$connected_peers}/{$expected_peers} Pairs connectées, cliquez pour plus de détails sur les ports P2P requis
```
Comment (nazar-pc) [inline res/translations/fr/messages.ftl:99]: Why are words `Nœud` and `Pairs` capitalized, is that really how it is supposed to be? They are not capitalized in English and many other languages.

**Review by nazar-pc — COMMENTED** [review]
Thanks, just one question

File: res/translations/fr/messages.ftl (hunk 1 above)
Comment (Vlady19) [reply res/translations/fr/messages.ftl:99]: No, not at all, but in France (often) when a new window appears (new subject) it is capitalized

Comment (nazar-pc) [reply res/translations/fr/messages.ftl:99]: Right, but the sentence starts with numbers in this case or with chain name on `running_node_title`

**Review by nazar-pc — APPROVED** [review]
Thanks!

**Pull request merged**